thoth/package_extract/image.py

```diff
@@ -281,6 +288,72 @@ def _run_apt_cache_show(
     return result
 
 
+def _get_lib_dir_symbols(root_dir, zeros=16):
+    to_ret = set({})
+    zero_string = "0" * zeros
```

Comment:
What are these zeros (please document them in sources)?

Reply:
Okay, I actually am changing it to just grep "0 A" because it is more robust but I'll document why